AutoCollection/HeartBeat.ts

```diff
@@ -94,26 +95,30 @@ class HeartBeat {
     }
 
     private _getAzureComputeMetadata(config: Config, callback: () => void) {
-        const metadataRequestUrl = `${HeartBeat.AIMS_URI}?${HeartBeat.AIMS_API_VERSION}&${HeartBeat.AIMS_FORMAT}`;
+        const metadataRequestUrl = `${AIMS_URI}?${AIMS_API_VERSION}&${AIMS_FORMAT}`;
         const requestOptions = {
             method: 'GET',
             headers: {
-                "Metadata": "true"
+                "Metadata": "true",
+                [AutoCollectHttpDependencies.disableCollectionRequestOption]: true,
```

Comment:
move this from `requestOptions.headers` to `requestOptions`

Reply:
I'll remove VM part in this PR but why move headers out to requestOptions? I see the interface RequestOptions has `headers?: { [key: string]: any };` property?
![image](https://user-images.githubusercontent.com/20292261/89329456-b0779000-d643-11ea-913c-e37ea966bb1a.png)

Reply:
only `[AutoCollectHttpDependencies.disableCollectionRequestOption]: true,` should be moved options. It should not be a header